bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/Canopus.java

```diff
@@ -329,13 +330,13 @@ public boolean inRange(ObsContext ctx, Offset offset){
         Option<SPTarget> spTargetOpt = getPrimaryCwfs3(ctx);
         if (spTargetOpt.isEmpty()) return None.instance();
 
-        SPTarget base   = ctx.getTargets().getBase();
-        SPTarget target = spTargetOpt.getValue();
+        Asterism asterism = ctx.getTargets().getAsterism();
+        SPTarget target   = spTargetOpt.getValue();
 
         final Option<Long> when = ctx.getSchedulingBlockStart();
 
         return
-            base.getSkycalcCoordinates(when).flatMap(bc ->
+            asterism.getSkycalcCoordinates(when).flatMap(bc ->
```

Comment:
It would be really nice to search and destroy the remaining uses of Skycalc coordinates.